api/end_device.proto

```diff
@@ -175,18 +173,79 @@ message EndDeviceVersion {
 }
 
 message MACSettings {
-  bool use_adr = 1 [(gogoproto.customname) = "UseADR"];
+  message DataRateIndexValue {
+    DataRateIndex value = 1;
+  }
+  message PingSlotPeriodValue {
+    PingSlotPeriod value = 1;
+  }
+  message AggregatedDutyCycleValue {
+    AggregatedDutyCycle value = 1;
+  }
+  message RxDelayValue {
+    RxDelay value = 1;
+  }
+
+  // Maximum delay for the device to answer a MAC request or a confirmed downlink frame.
+  // Must be set if the device supports class B.
```

Comment:
@htdvisser @johanstokking do we need to enforce this? BE says we should, but it looks like we could also just use a NS default?